README.md

```diff
@@ -131,6 +132,9 @@ const perfume = new Perfume({
       case 'tbtFinal':
         myAnalyticsTool.track('totalBlockingTimeFinal', { duration: data });
         break;
+      case 'page-title':
```

Comment:
Because this is part of an Element Timing metric, let's try to make it more simple for any newcomer to understand what this metric is for. We could say `elPageTitle` then in the tracking `elementTimingPageTitle`. In the end, people will choose the name they prefer in their app, but here in the quickstart, we need to find names that are self-explanatory.

Reply:
Agreed - made it more obvious and used it consistently in: [https://github.com/Zizzamia/perfume.js/pull/124/commits/dd7d7acfbe2cfd8da4dc4c391fe38b22d36b895e ](https://github.com/Zizzamia/perfume.js/pull/124/commits/dd7d7acfbe2cfd8da4dc4c391fe38b22d36b895e)